apps/rule-manager/client/src/ts/components/context/tags.tsx

```diff
@@ -163,9 +163,9 @@ export const TagsProvider: React.FC = ({ children }) => {
 		}
 	};
 
-	useEffect(() => {
-		fetchTags();
-	}, []);
+  useEffect(() => {
+    fetchTags()
+  }, [])
```

Comment:
This 'Move tag state into context' commit seems to just be formatting?

Reply:
This is probably the result of a rebase that didn't include formatting changes upstream. My fault for getting ahead of myself and producing several linked branches in quick succession, there are lots of rebases involved! The original commit is in https://github.com/guardian/typerighter/pull/430/commits/dd22b8e2ed0734ef14ef475fc6e1ec9f3ef49a8f if it's useful.